iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -59,6 +75,10 @@ internal sealed class HttpClientHelper
                 _httpClientHandler.Proxy = proxy;
             }
 
+            _httpClientHandler.MaxConnectionsPerServer = DefaultMaxConnectionsPerServer;
+            ServicePoint servicePoint = ServicePointManager.FindServicePoint(_baseAddress);
+            servicePoint.ConnectionLeaseTimeout = DefaultConnectionLeaseTimeout.Milliseconds;
```

Comment:
Do you see any value in us making this configurable?

Reply:
I'd rather we just allow users to pass in the entire HttpClient if they want to configure settings at this level

Reply:
That's fair. Remind me, do we configure the user-supplied http client in any way, or use it as-is?

Reply:
We use it as-is. However, we currently only allow users to pass in a HttpClient in our IoT hub service client, so we'll need to do that work in all the other clients still.